composer.json

```diff
@@ -32,6 +32,7 @@
 	"require-dev": {
 		"craue/translations-tests": "^1.0",
 		"doctrine/common": "~2.7",
+		"doctrine/dbal": "<3.0",
```

Comment:
Please revert this change and rebase to current master. I've added support for DBAL 3.